dune/grid/cpgrid/OrientedEntityTable.hpp

```diff
@@ -162,6 +164,36 @@ namespace Dune
 		return super_t::operator==(other);
 	    }
 
+            /** @brief Prints the relation matrix corresponding to the table, sparse format.
+
+             Let the entities of codimensions f and t be given by
+             the sets \f$E^f = { e^f_i } \f$ and \f$E^t = { e^t_j }\f$.
+             A relation matrix R is defined by
+	     \f{eqnarray*}{
```

Comment:
Does Doxygen not support anything more well-behaved than `eqnarray`?

If I were writing this in a regular document I'd use something like
```latex
\usepackage{amsmath}
% ...
\begin{document}
% ...
\[
R_{ij} =
\begin{cases}
  \phantom{-} 0, &\quad \text{if } e_i^f \text{ and } e_j^f \text{ are not neighbours}, \\
  \phantom{-} 1, &\quad \text{if } e_i^f \text{ and } e_j^f \text{ are neighbours with same orientation}, \\
           -  1, &\quad \text{if } e_i^f \text{ and } e_j^f \text{ are neighbours with opposite orientation}.
\end{cases}
\]
```
Is that not possible in Doxygen?